src/software/utils/main_imageProcessing.cpp

```diff
@@ -151,31 +208,8 @@ int aliceVision_main(int argc, char * argv[])
         image::readImage(view.getImagePath(), image, image::EImageColorSpace::LINEAR);
         oiio::ParamValueList metadata = image::readImageMetadata(view.getImagePath());
 
-        unsigned int nchannels = 3;
-
-        if (downscale != 1.0f)
-        {
-            const unsigned int w = view.getWidth();
-            const unsigned int h = view.getHeight();
-            const unsigned int nw = (unsigned int)(floor(float(w) / downscale));
-            const unsigned int nh = (unsigned int)(floor(float(h) / downscale));
-
-            image::Image<image::RGBfColor> rescaled(nw, nh);
-
-            const oiio::ImageSpec imageSpecResized(nw, nh, 3, oiio::TypeDesc::FLOAT);
-            const oiio::ImageSpec imageSpecOrigin(w, h, 3, oiio::TypeDesc::FLOAT);
-            oiio::ImageBuf bufferOrigin(imageSpecOrigin, image.data());
-            oiio::ImageBuf bufferResized(imageSpecResized, rescaled.data());
-            oiio::ImageBufAlgo::resample(bufferResized, bufferOrigin);
-
-            const oiio::ImageBuf inBuf(oiio::ImageSpec(w, h, nchannels, oiio::TypeDesc::FLOAT), image.data());
-            oiio::ImageBuf outBuf(oiio::ImageSpec(nw, nh, nchannels, oiio::TypeDesc::FLOAT), rescaled.data());
-
-            oiio::ImageBufAlgo::resize(outBuf, inBuf);
-
-            image.swap(rescaled);
-        }
-        if (exposureCompensation)
+        // if exposureCompensation is needed for sfmData fIles
```

Comment:
`fIles` minor typo

Reply:
Fixed in https://github.com/alicevision/AliceVision/pull/806/commits/4670526623ce19e5c00f8cd128f8eaebc9acd934 .